### ISSUE TITLE
Sudoku: focus styles missing for controls outside of the board itself.

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#191]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/sudoku/

For the most part they are input type="submit". It looks like their custom styling don't give focus styles. Changing to buttons and using our default styles may be an option here.

Selects are also somewhat broken, missing arrow and focus styles.

**Comment by dstorey** [comment]
May also want the minimal header
